### DISCUSSION TITLE
Vietnam IT Salary Guide 2024

### DISCUSSION THREAD

**Discussion by huyle250502** [community/community#80865] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

While networking at tech events, it's also essential to have a solid grasp of industry insights, including salary trends. I recently found a Vietnam IT salary guide that's been an eye-opener. If you'd like to know more, you can see the following link: https://bit.ly/47OaxVH

**Comment by huyle250502** [comment]
Explore the Vietnam IT Salary Guide 2024 to understand:

- How economic challenges are impacting the IT HR market
- What to expect in 2024 and how to prepare
- Salary insights that can be your compass in turbulent times

For personalized guidance and to discuss how we can assist you, visit: https://bit.ly/47OaxVH

**Comment by Akash1134** [comment]
Hi @huyle250502 , we’re glad you’re here 😀

Thanks for sharing your post with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community on Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
